gui/storage/models.py

```diff
@@ -322,8 +322,7 @@ class Disk(Model):
         editable=False,
         default=1,
     )
-    disk_serial = models.CharField(
-        max_length=30,
+    disk_serial = models.TextField(
```

Comment:
I was thinking of ignoring the value sent on POST so we dont have to change the model which will grant a migration.